public/locales/en/common.json

```
@@ -243,34 +243,54 @@
     ]
   },
   "mentors": {
-    "title": "Mentors & Jurors"
+    "title": "Mentors & Jurors",
+    "roles": {
+      "jury": "Jury",
+      "mentor": "Mentor",
+      "speaker": "Speaker"
+    }
   },
   "schedule": {
     "title": "Schedule",
+    "info": "Details of the schedule are subject to change.",
     "days": [
       {
         "name": "04.11 - FRIDAY",
         "eventNames": {
-          "hackatonStart": "Hackaton starts",
-          "javaLecture": "Something about Java",
-          "graphQLforBegginers": "GraphQL Lecture",
-          "rememberToBeHydrated": "Time for drinking water 🍼"
+          "hackatonStart": "Start of the event",
+          "firstBreak": "Break",
+          "serviceLevelObjectives": "Lecture: Łukasz Dobek from Nobl9 - „Service Level Objectives”",
+          "secondBreak": "Break",
+          "modernAppPerformance": "Lecture: Piotr Jabłoński from WithSecure - „Performance of modern web applications - why bother with kilobytes in 2022?”",
+          "thirdBreak": "Break",
+          "accessibilityInApps": "Lecture: Jan Śmiełowski, Mikołaj Szymczak from All for one - „About accessibility, or how to write an application that is friendly to every user”",
+          "fourthBreak": "Break",
+          "hackatonGuide": "Lecture by Robert Banaszak from Talkie.ai - „Hackathon survival guide, or how to get the most value out of a hackathon”",
+          "endOfDay": "Closing of the day"
         }
       },
       {
         "name": "05.11 - SATURDAY",
         "eventNames": {
-          "pizzaForBreakfast": "Breakfast time 🍕",
-          "whyDjangoOnHackatonIsNotTheBestIdea": "Something about web programming in Python",
-          "pizzaForLunch": "Lunch time 🍕",
-          "pizzaForDinner": "Dinner time 🍕"
+          "morningStart": "Coyote morning",
+          "breakfast": "Breakfast",
+          "logicApp": "Lecture: Dawid Chróścielski from AJTiTi - „Logic App and Visual Programming”",
+          "serverless": "Lecture: Kacper Chomicz from WTT - „Modern Application Architectures - Serverless in the Cloud”",
+          "lunchBreak": "Lunch",
+          "dancingTime": "Chapelloise",
+          "agileLecture": "Keynote speaker: Joanna Zakrocka from Inetum - „Mind in Agile mode”",
+          "timeForSubmitting": "Time for submitting projects",
+          "endOfSubmitting": "End of submission of projects",
+          "projectsPresentations": "Projects presentations",
+          "closingCeremony": "Closing ceremony and announcement of winners"
```

We could reduce some `of` if possible
```suggestion
          "closingCeremony": "Closing ceremony and winners announcement"
```